core/src/main/java/org/jdbi/v3/mapper/BuiltInMapperFactory.java

```diff
@@ -163,4 +169,15 @@ private static ZonedDateTime getZonedDateTime(ResultSet r, int i) throws SQLExce
         Timestamp ts = r.getTimestamp(i);
         return ts == null ? null : ZonedDateTime.ofInstant(ts.toInstant(), ZoneId.systemDefault());
     }
+
+    private static OffsetTime getOffsetTime(ResultSet r, int i) throws SQLException {
+        Time time = r.getTime(i);
+        ZoneOffset zoneOffset = Instant.now().atZone(ZoneId.systemDefault()).getOffset();
```

Comment:
That's a valid concern. We could probably support only `LocalTime`, because  `OffsetTime` is to ambiguous.
We have no means to store/retrieve the time zone, because JDBC doesn't provide such API.